sdk/core/azure-core-amqp/test/ut_uamqp/uamqp_value_tests.cpp

```diff
@@ -222,18 +221,19 @@ TEST_F(TestValue, SimpleCreate)
   }
 
   {
-    AmqpValue descriptor(235ll);
+    AmqpValue descriptor(static_cast<uint64_t>(237ll));
```

Comment:
`237ULL`? (+ there is at least one similar case below)
Static cast is probably also not necessary if `ULL` is used.
(`ULL` is C++14, which we can use - https://en.cppreference.com/w/cpp/language/integer_literal)

Reply:
static case is necessary. If you think it's not, take it up with the clang developers :).